### ISSUE TITLE
Various tests started timing out on v2.2.x

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#5365]
A few of them:
```
TIME changefeeds.basic
TIME clustering.single_server_failure
TIME error_tolerant.kill-access
TIME error_tolerant.kill-secondary
TIME interface.emergency_repair
```

These tests are still passing for me on `v2.2.3-1`, but no longer on `v2.2.x` (each time using the same server binary from `v2.2.x`).

It looks like they started timing out with https://github.com/rethinkdb/rethinkdb/commit/f1c91f58aa37293423310c283c650d81316875ee
Possibly related to https://github.com/rethinkdb/rethinkdb/issues/5363 ?

**Comment by danielmewes** [comment]
I might need to revert f1c91f5 temporarily until @larkost gets back.

**Comment by danielmewes** [comment]
... The `changefeeds.basic` one is probably a red herring. I think the other ones might be failing for the same reason as #5363 , since they rely on shutting down / killing servers as part of the testing process.

**Comment by danielmewes** [comment]
@larkost I've temporarily reverted the commit in v2.2.x with d261c03 . Please re-apply once we know what's going wrong with those tests.

**Comment by larkost** [comment]
The issue behind this was fixed in `next` with 4f8aa7d then re-synced with `v2.2.x` with 51afc02.
